qemu/tests/win_virtio_driver_install_from_update.py

```diff
@@ -89,5 +100,12 @@ def start_wuauserv_service(session):
         test.fail(fail_log)
     elif "TRUE" not in chk_output:
         test.error("Device %s is not found in guest" % device_name)
+    cmd = wmic.make_query(
```

Comment:
I'd like to recommend using the present utility.
https://github.com/autotest/tp-qemu/blob/master/provider/win_driver_utils.py#L62

Reply:
update it, thanks